src/main/java/core/basesyntax/db/FruitShopDao.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.db;
+
+import java.util.List;
+
+public interface FruitShopDao {
+    int storageSize();
+
+    List<String> getKeyAndValue();
+
+    void put(String fruit, int quantity);
+
+    void putOnlyFruits(String fruit);
```

Comment:
The method 'putOnlyFruits' is not clear in its purpose. If it's meant to add a fruit without a quantity, consider renaming it to 'addFruitWithoutQuantity' for clarity.